services/user-svc/internal/organization/organization.go

```diff
@@ -715,8 +695,27 @@ func (s ServiceServer) RevokeInvitation(ctx context.Context, req *pb.RevokeInvit
 }
 
 func CreateOrganizationAndAddUser(ctx context.Context, attr organization_repo.Organization, userID uuid.UUID) (*organization_repo.Organization, error) {
```

Comment:
re-using a db-related struct for this is likely going to cause headaches in the future and was a bad idea, just pass all relevant values as single parameters or create a new struct for this

Reply:
Yes. The `user-svc` is about to be rewritten. Then we tackle such issues.